frontend/docs/pages/home/features/concurrency/cancel-in-progress.mdx

```diff
@@ -37,15 +37,16 @@ from hatchet_sdk import ConcurrencyLimitStrategy
 
 @hatchet.workflow(on_events=["concurrency-test"])
 class ConcurrencyDemoWorkflow:
-    @hatchet.concurrency(max_runs=10, limit_strategy=ConcurrencyLimitStrategy.CANCEL_IN_PROGRESS)
-    def concurrency(self, context) -> str:
-        return context.workflow_input()["user_id"]
+@hatchet.concurrency(max_runs=10, limit_strategy=ConcurrencyLimitStrategy.CANCEL_IN_PROGRESS)
+def concurrency(self, context) -> str:
```

Comment:
looks like we lost tabs here?

Reply:
I think the lint:fix sometimes actually breaks things.. this happened to me before. Might be a tricky one to resolve

Reply:
yeah, lint:fix is breaking things up